tests/_data/stores.php

```diff
@@ -0,0 +1,18 @@
+<?php
```

Comment:
This file is also obsolete, if you remove the vfsStream.

Reply:
removed